packages/analytics-js/src/components/configManager/util/validate.ts

```diff
@@ -30,4 +31,6 @@ const isValidSourceConfig = (res: any): boolean =>
   isObjectLiteralAndNotNull(res.source.config) &&
   Array.isArray(res.source.destinations);
 
-export { validateLoadArgs, isValidSourceConfig };
+const isValidStorageType = (storageType?: StorageType): boolean => typeof storageType === 'string';
```

Comment:
We must also check whether the value is precisely one of the accepted values.

Reply:
Are you suggesting changing the StorageType to an enum instead type?

Reply:
Done